segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/handler/PravegaRequestProcessor.java

```diff
@@ -485,6 +490,18 @@ public void createSegment(CreateSegment createStreamSegment) {
                 });
     }
 
+    private CompletableFuture<Void> createIndexSegment(final String segmentName) {
+        Collection<AttributeUpdate> attributes = Arrays.asList(
+                new AttributeUpdate(CREATION_TIME, AttributeUpdateType.None, System.currentTimeMillis()),
+                new AttributeUpdate(ATTRIBUTE_SEGMENT_TYPE, AttributeUpdateType.None, SegmentType.INDEX_SEGMENT.getValue()),
+                new AttributeUpdate(EVENT_COUNT, AttributeUpdateType.Accumulate, 0L), // to store the number of events
+                new AttributeUpdate(ALLOWED_INDEX_SEG_EVENT_SIZE, AttributeUpdateType.Replace, 0L)
```

Comment:
Will it be fixed for this child segment? If yes then AttributeUpdateType can be None.

Reply:
Addressed review comment